t/16-utils.t

```diff
@@ -502,4 +503,17 @@ subtest 'human readable size' => sub {
     is(human_readable_size(245760), '240 KiB', 'less than a MB');
 };
 
+subtest 'create downloads list' => sub {
+    my $job_settings = {};
+    is_deeply create_downloads_list($job_settings), {}, 'empty for no settings provided';
+    $job_settings->{TEST_URL} = 'test';
+    $job_settings->{TEST} = 'test';
+    is_deeply create_downloads_list($job_settings), {}, '';
+    $job_settings->{ISO_URL} = 'test';
+    $job_settings->{ISO} = 'test';
+    my $utils_mocked = Test::MockModule->new('OpenQA::Utils');
+    $utils_mocked->redefine(locate_asset => 'foo/bar');
+    is_deeply create_downloads_list($job_settings), {test => ['foo/bar', '0']}, '';
```

Comment:
I would write it as this (tested)

```suggestion
    is_deeply create_downloads_list({}), {}, 'empty for no settings provided';
    is_deeply create_downloads_list({TEST_URL => 'test', TEST => 'test'}), {}, '';
    my $utils_mocked = Test::MockModule->new('OpenQA::Utils');
    $utils_mocked->redefine(locate_asset => 'foo/bar');
    is_deeply create_downloads_list({ISO_URL => 'test', ISO => 'test'}), {test => ['foo/bar', '0']}, '';
```

Reply:
Good suggestion, that looks much better. Thanks.